.changes/unreleased/🎉 Feature-20230116-183844.yaml

```diff
@@ -0,0 +1,8 @@
+kind: "\U0001F389 Feature"
+body: Add sort flag to client search command.
```

Comment:
same as other PR feedback, adjust the changelog and tree to be a little bit more complete since it's target is changelog entires that would be read by users.

Reply:
Do you mean I need to explain what sort flag does? It can be read in the command usage